spec/features/lettings_log_spec.rb

```diff
@@ -413,5 +413,24 @@
         expect(page).to have_current_path("/lettings-logs")
       end
     end
+
+    context "when there are duplicate logs" do
+      let(:lettings_log) { create(:lettings_log, :completed, owning_organisation: user.organisation, created_by: user) }
+      let!(:duplicate_log) do
+        duplicate = lettings_log.dup
+        duplicate.id = nil
+        duplicate.save!
+        duplicate
+      end
+
+      it "is possible to delete duplicate logs" do
+        visit lettings_log_delete_duplicates_path(lettings_log)
+        click_button "Delete this log"
+        duplicate_log.reload
+        expect(duplicate_log.status).to eq("deleted")
+        expect(page).to have_css(".govuk-notification-banner.govuk-notification-banner--success")
+        expect(page).to have_content("Log #{duplicate_log.id} has been deleted")
+      end
+    end
```

Comment:
is the plan to add to this feature test to encompass the earlier steps that are part of Aaron's ticket once that's done?

Reply:
Hopefully!